### PR TITLE
Feat: Adds tokens to `EmptyState`

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `EmptyState`: new scoped tokens linked to global ones.

## How does it work?

This PR uses local variables to parameterize the `EmptyState` properties, then connects these scoped tokens to the global ones.

### New local tokens for `EmptyState`

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-empty-state-height`|`100%`|
|`--fs-empty-state-min-height`|`50vh`|
|`--fs-empty-state-padding`|`0 var(--fs-spacing-8)`|
|`--fs-empty-state-bkg-color`|`var(--fs-color-neutral-bkg)`|

### Rounded Variant
| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-empty-state-border-radius`|`var(--fs-border-radius)`|

### Related Components

|EmptyCart|EmptyGallery|
|-|-|
|||

### Title

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-empty-state-title-margin-bottom`|`var(--fs-spacing-2)`|
|`--fs-empty-state-title-color`|`var(--fs-color-disabled-text)`|
|`--fs-empty-state-title-text-size`|`var(--fs-text-size-3)`|
 
### Link

| Local token                                 | Default value/Global token linked          |
| - | - |
|`--fs-empty-state-link-min-width`|`11.875rem`|

## How to test it?

- The `EmptyState` component and its related components should look just as it was before these changes.
- You can check the story on Storybook.

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.
- [ ] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

File: src/components/cart/EmptyCart/EmptyCart.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<header>
+
+# EmptyCart
+
+An implementation of the `EmptyState` that represents the Cart with no items.
+
+</header>
+
+---
+
```
Comment (hellofanny) [inline src/components/cart/EmptyCart/EmptyCart.stories.mdx:20]:
```suggestion

```

File: src/components/ui/EmptyState/empty-state.module.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  // --------------------------------------------------------
+  --fs-empty-state-height               : 100%;
+  --fs-empty-state-min-height           : 50vh;
+  --fs-empty-state-padding              : 0 var(--fs-spacing-8);
+  --fs-empty-state-bkg-color            : var(--fs-color-neutral-bkg);
+
+  // Title
+  --fs-empty-state-title-margin-bottom : var(--fs-spacing-2);
+  --fs-empty-state-title-color         : var(--fs-color-disabled-text);
+  --fs-empty-state-title-text-size     : var(--fs-text-size-3);
```
Comment (hellofanny) [inline src/components/ui/EmptyState/empty-state.module.scss:15]:
```suggestion
  --fs-empty-state-title-text-size     : var(--fs-text-size-lead);
```

File: src/components/ui/EmptyState/EmptyState.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import { TokenTable, TokenRow } from 'src/../.storybook/components'
+
+import EmptyState from '.'
+import EmptyGallery from 'src/components/sections/ProductGallery'
+import EmptyCart from 'src/components/cart/EmptyCart'
+
+<Meta component={EmptyState} title="Organisms/EmptyState/Default" />
+
+export const Template = (args) => (
+  <EmptyState {...args}>The list is empty</EmptyState>
```
Comment (hellofanny) [inline src/components/ui/EmptyState/EmptyState.stories.mdx:12]: Hmm, how about `Nothing to display`? 🤔

File: src/components/ui/EmptyState/EmptyState.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+---
+
+## Related Components
+
+<section className="sbdocs-section">
+  <article>
+    <div style={{ height: '250px' }}>
+      <a href="?path=/docs/organisms-emptystate-emptycart--default-story">
+        <img width="245px" src="https://vtexhelp.vtexassets.com/assets/docs/src/empty-car___33a48f795ba03130f1bc81f1c075143d.png"/>
```
Comment (hellofanny) [inline src/components/ui/EmptyState/EmptyState.stories.mdx:67]:
```suggestion
        <img width="190px" src="https://vtexhelp.vtexassets.com/assets/docs/src/empty-car___33a48f795ba03130f1bc81f1c075143d.png"/>
```

Just to look more even 🙇 

image

File: src/components/cart/EmptyCart/EmptyCart.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+export const Template = (args) => <EmptyCart {...args}></EmptyCart>
+
+<header>
+
+# EmptyCart
+
+An implementation of the `EmptyState` that represents the Cart with no items.
+
+</header>
+## Usage
```
Comment (filipewl) [inline src/components/cart/EmptyCart/EmptyCart.stories.mdx:18]: Noticed the other stories had this separator.

```suggestion
</header>

---

## Usage
```

File: src/components/ui/EmptyState/EmptyState.stories.mdx (hunk 3 above)
Comment (eduardoformiga) [reply src/components/ui/EmptyState/EmptyState.stories.mdx:12]: Nice!

File: src/components/ui/EmptyState/EmptyState.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  <article>
+    <div style={{ height: '250px' }}>
+      <a href="?path=/docs/organisms-emptystate-emptycart--default-story">
+        <img width="190px" src="https://vtexhelp.vtexassets.com/assets/docs/src/empty-car___33a48f795ba03130f1bc81f1c075143d.png"/>
+      </a>
+    </div>
+    <article className="sbdocs-section-text">
+      <h3>Empty Cart</h3>
+      <p>
+        Displays the empty cart message along with the start shopping button.
```
Comment (hellofanny) [inline src/components/ui/EmptyState/EmptyState.stories.mdx:71]:
```suggestion
        Displays the empty cart message along with the `Start Shopping` button.
```

**Review by renatamottam — COMMENTED** [review]
I have some suggestions for the doc organization, we can talk later :)

**Comment by eduardoformiga** [comment]
> I have some suggestions for the doc organization, we can talk later :)

I implemented it [here](https://github.com/vtex-sites/nextjs.store/pull/125). Thanks!

**Pull request merged**